doc/source/examples/unresolved-cfd-dem/liquid-solid-fluidized-bed/liquid-solid-fluidized-bed.rst

```diff
@@ -0,0 +1,500 @@
+
+Saffman lift force is proven to be very important to properly reproduce particles' dynamics in the liquid-fluidized bed `[1] <https://doi.org/10.1016/j.powtec.2023.118652>`_.
+
+.. code-block:: text
+
+    subsection cfd-dem
+      set vans model         = modelA
+      set grad div           = true
+      set drag model         = rong
+      set buoyancy force     = true
+      set shear force        = true
+      set pressure force     = true
+      set saffman lift force = true
+      set coupling frequency = 100
+      set void fraction time derivative = false
+    end
+
+.. warning::
+    Void-fraction time-derivative lead to significant instability in the case of liquid-fluidized beds, hence we do not use it.
+
+Non-linear Solver
+~~~~~~~~~~~~~~~~~
+
+We use the inexact Newton non-linear solver to minimize the number of time the matrix of the system is assembled. This is used to increase the speed of the simulation, since the matrix assembly requires significant computations.
+
+.. code-block:: text
+
+  subsection non-linear solver
+    set solver           = inexact_newton
+    set tolerance        = 1e-10
+    set max iterations   = 10
+    set verbosity        = verbose
+  end
+
+Linear Solver
+~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection linear solver
+      set method                                = gmres
+      set max iters                             = 5000
+      set relative residual                     = 1e-3
+      set minimum residual                      = 1e-11
+      set ilu preconditioner fill               = 1
+      set ilu preconditioner absolute tolerance = 1e-14
+      set ilu preconditioner relative tolerance = 1.00
+      set verbosity                             = verbose
+    end
+
+
+------------------------------
+Running the CFD-DEM Simulation
+------------------------------
+
+The simulation is run (on 8 core) using the ``cfd_dem_coupling`` application as following:
+
+.. code-block:: text
+
+    mpirun -np 8 cfd_dem_coupling dem-packing-in-fluidized-bed.prm
+
+.. note::
+    Running the packing of alumina particles should take approximately 4 hours and 15 minutes on 16 cores.
+
+--------
+Results
+--------
+
+We briefly comment on some results that can be extracted from this example.
+
+.. important::
+
+    This example includes a post-processing file written in Python and using `lethe_pyvista_tools <../../dem/rotating-drum/small-scale-rotating-drum-post-processing.html>`_. module.
+
+    To use the code, run ``python3 lsfb_postprocessing.py $PATH_TO_YOUR_CASE_FOLDER``. The code will generate several graphics showing the pressure profile within the bed, which are going to be stored in ``$PATH_TO_YOUR_CASE_FOLDER/P_x``. It will also generate a ``deltaP_t.csv`` files with the total pressure difference for each time-step. Additionally, it generates a void fraction as a function of time graphic (``eps_t.png``).
```

Comment:
```suggestion
    To use the code, run ``python3 lsfb_postprocessing.py $PATH_TO_YOUR_CASE_FOLDER``. The code will generate several graphics showing the pressure profile within the bed, which are going to be stored in ``$PATH_TO_YOUR_CASE_FOLDER/P_x``. It will also generate a ``deltaP_t.csv`` file with the total pressure difference for each time-step. Additionally, it generates a void fraction as a function of time graphic (``eps_t.png``).
```